### PR TITLE
matching each regex against every line in description separately

### LINKED ISSUE #31: Description configuration should allow matching per line
format->description should be optional unbounded list of reges expressions per line. We shouldn't require that the whole description is matching the regex passed in the config.

### PR DESCRIPTION
**Pull request by xjusko**
This fixes #31.
I have implemented it so at least one line in the description has to match at least one pattern in the config file.

### CODE REVIEW

**Review by xstefank — CHANGES_REQUESTED** [review]
Why not allowing custom message per regex?

**Comment by xstefank** [comment]
Yes, it should return the message for the line or you can optionally define a top level message default, or the fallback default if none of these two is configured.

**Comment by xstefank** [comment]
@xjusko needs rebase

File: src/main/java/io/xstefank/wildlfy/bot/config/Format.java [hunk 1]
```diff
@@ -7,7 +7,8 @@ public final class Format {
     @JsonProperty("title-check")
     public RegexDefinition titleCheck;
 
-    public RegexDefinition description;
+    @JsonProperty("description")
```
Comment (xstefank) [inline src/main/java/io/xstefank/wildlfy/bot/config/Format.java:10]: why do you need `@JsonProperty` here? Name is automatically deduced from the field name.

**Review by xstefank — APPROVED** [review]
One minor comment, merging, but please explain.

File: src/main/java/io/xstefank/wildlfy/bot/config/Format.java (hunk 1 above)
Comment (xjusko) [reply src/main/java/io/xstefank/wildlfy/bot/config/Format.java:10]: Sorry, I forgot to remove it.

**Pull request merged**